translate/storage/aresource.py

```diff
@@ -31,12 +31,219 @@
 from translate.storage import base, lisa
 from translate.misc.multistring import multistring
 
-from babel.core import Locale, UnknownLocaleError
-
 EOF = None
 WHITESPACE = ' \n\t'  # Whitespace that we collapse.
 MULTIWHITESPACE = re.compile('[ \n\t]{2}(?!\\\\n)')
 
+# List of plural tags generated from CLDR 32.0.1 using
+# https://github.com/WeblateOrg/language-data
+PLURAL_TAGS = {
```

Comment:
Please put this on `translate.lang.data` besides all other language-related data.